### ISSUE TITLE
Windows Defender находит Trojan:Win32/Wacatac.H!ml

### ISSUE THREAD

**Issue by Serlutin** [alexgrim123/botcRuScripter#1]
This program is dangerous and executes commands from an attacker.

Хотя версия с tesera вроде бы защитник не триггерила.

(понятно, что совет: удоли винду и защитник, но, к сожалению, не получится программу посоветовать кому-то)

**Comment by alexgrim123** [comment]
Да, это классический трабл гамака (первой студии по крайней мере), всё скомпилированное считается вирусом почему-то. К сожалению с этим ничего не поделаешь :(.

**Comment by Serlutin** [comment]
А можно с вами в дискорде списаться? у меня там ник такой же как здесь
